File: src/main/java-graphql/io/github/graphql/java/model/MetaQueryResponse.java
package io.github.graphql.java.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult;
import java.util.Map;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-07T19:46:14+0800"
)
public class MetaQueryResponse extends GraphQLResult<Map<String, GitHubMetadataTO>> {

    private static final String OPERATION_NAME = "meta";

    public MetaQueryResponse() {
    }

    public GitHubMetadataTO meta() {
        Map<String, GitHubMetadataTO> data = getData();
        return data != null ? data.get(OPERATION_NAME) : null;
    }

}

File: src/main/scala-graphql/io/github/graphql/model/MetaQueryResponse.scala
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult
import java.util.{ Map => JMap }

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:01+0800"
)
class MetaQueryResponse extends GraphQLResult[JMap[String, GitHubMetadataTO]] {

    def meta(): GitHubMetadataTO = {
        val data: JMap[String, GitHubMetadataTO] = getData
        if (data != null) data.get(MetaQueryResponse.OPERATION_NAME) else null
    }

}

object MetaQueryResponse {

    private final val OPERATION_NAME: String = "meta"

}
